### PR TITLE
56-no-support-for-ingestion-policies

### PR DESCRIPTION
**Pull request by snltd**
Adds an `ingestion_policy` resource.

Bumps `version` to 2.8.0, but I'm not sure if that file is actually used to create the version. I think 2.8.0 is correct, as this adds a new feature without breaking any compatibility.

Passed `make testacc` on my machine, running against `longboard.wavefront.com`.

### CODE REVIEW

File: go.sum [hunk 1]
```diff
@@ -35,6 +35,7 @@ github.com/bgentry/speakeasy v0.1.0/go.mod h1:+zsyZBPWlz7T6j88CTgSN5bM796AkVf0kB
 github.com/census-instrumentation/opencensus-proto v0.2.1/go.mod h1:f6KPmirojxKA12rnyqOA5BBL4O983OfeGPqjHWSTneU=
```
Comment (keep94) [inline go.sum:34]: Did you remember to run `go mod tidy`?

File: wavefront/resource_ingestion_policy.go [hunk 2]
```diff
@@ -59,7 +59,7 @@ func resourceIngestionPolicyRead(
 	d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*wavefrontClient).client.IngestionPolicies()
 	id := d.Id()
-	policy := wavefront.IngestionPolicy{ID: &id}
+	policy := wavefront.IngestionPolicy{ID: id}
```
Comment (keep94) [inline wavefront/resource_ingestion_policy.go:62]: Nit: You could just do `policy := wavefront.IngestionPolicy{ID: d.Id()}`

File: wavefront/resource_ingestion_policy.go [hunk 3]
```diff
@@ -59,7 +59,7 @@ func resourceIngestionPolicyRead(
 	d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*wavefrontClient).client.IngestionPolicies()
 	id := d.Id()
```
Comment (keep94) [inline wavefront/resource_ingestion_policy.go:61]: I don't think you need the id variable since you don't need a string pointer anymore.

**Review by keep94 — CHANGES_REQUESTED** [review]
Thank you for doing all this work Robert. While you are at it could you do one more small thing? The releases page link under the "Installing the Plugin" section of README.md is wrong.  It should be `https://github.com/vmware/terraform-provider-wavefront/releases` not `https://github.com/WavefrontHQ/terraform-provider-wavefront/releases`

**Comment by snltd** [comment]
> Thank you for doing all this work Robert. While you are at it could you do one more small thing? The releases page link under the "Installing the Plugin" section of README.md is wrong. It should be `https://github.com/vmware/terraform-provider-wavefront/releases` not `https://github.com/WavefrontHQ/terraform-provider-wavefront/releases`

Done. Also fixed the part that tells you how to run acceptance tests, which was wrong.

**Review by keep94 — APPROVED** [review]
Looks good to me

**Comment by keep94** [comment]
snltd,

I have released 2.8.0 of the wavefront terraform plugin, but I forgot to have you update the terraform plugin documentation for your new ingestion_policy_resource, and I forgot to have you update the documentation for service accounts.  I plan to update the documentation myself, but I have one question for you.  What goes in the ingestion_policy field of the wavefront_service_account resource?  The name of the ingestion policy or the ID?

**Pull request merged**